File: sample/src/main/java/com/afollestad/aestheticsample/SecondaryFragment.java
package com.afollestad.aestheticsample;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import butterknife.ButterKnife;
import butterknife.OnClick;
import butterknife.Unbinder;

/** @author Aidan Follestad (afollestad) */
public class SecondaryFragment extends Fragment {

  private Unbinder unbinder;

  @Nullable
  @Override
  public View onCreateView(
      LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
    return inflater.inflate(R.layout.fragment_secondary, container, false);
  }

  @Override
  public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
    super.onViewCreated(view, savedInstanceState);
    unbinder = ButterKnife.bind(this, view);
  }

  @Override
  public void onDestroyView() {
    unbinder.unbind();
    super.onDestroyView();
  }

  @OnClick(R.id.drawer_layout)
  public void onClickDrawerLayout() {
    startActivity(new Intent(getActivity(), DrawerActivity.class));
  }

  @OnClick(R.id.coordinator_layout)
  public void onClickCoordinatorlayout() {
    startActivity(new Intent(getActivity(), CoordinatorLayoutActivity.class));
  }

  @OnClick(R.id.bottom_tabs)
  public void onClickBottomTabs() {
    startActivity(new Intent(getActivity(), BottomNavActivity.class));
  }

  @OnClick(R.id.recycler_view)
  public void onClickRecyclerView() {
    startActivity(new Intent(getActivity(), RecyclerViewActivity.class));
  }
}

File: sample/src/main/java/com/afollestad/aestheticsample/SecondaryFragment.kt
package com.afollestad.aestheticsample

import android.content.Intent
import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.fragment_secondary.view.bottom_tabs
import kotlinx.android.synthetic.main.fragment_secondary.view.coordinator_layout
import kotlinx.android.synthetic.main.fragment_secondary.view.drawer_layout
import kotlinx.android.synthetic.main.fragment_secondary.view.recycler_view

/** @author Aidan Follestad (afollestad)
 */
class SecondaryFragment : Fragment() {

  override fun onCreateView(
    inflater: LayoutInflater,
    container: ViewGroup?,
    savedInstanceState: Bundle?
  ): View? {
    return inflater.inflate(R.layout.fragment_secondary, container, false)
  }

  override fun onViewCreated(
    view: View,
    savedInstanceState: Bundle?
  ) {
    super.onViewCreated(view, savedInstanceState)

    view.drawer_layout.setOnClickListener {
      startActivity(
          Intent(activity, DrawerActivity::class.java)
      )
    }
    view.coordinator_layout.setOnClickListener {
      startActivity(
          Intent(activity, CoordinatorLayoutActivity::class.java)
      )
    }
    view.bottom_tabs.setOnClickListener {
      startActivity(
          Intent(activity, BottomNavActivity::class.java)
      )
    }
    view.recycler_view.setOnClickListener {
      startActivity(
          Intent(activity, RecyclerViewActivity::class.java)
      )
    }
  }
}
